src/packages/block/block/workspace/block-workspace.context.ts

```diff
@@ -277,7 +293,10 @@ export class UmbBlockWorkspaceContext<LayoutDataType extends UmbBlockLayoutBaseM
 	}
 
 	getName() {
-		return 'block name content element type here...';
+		return this.#label.value;
+	}
+	setName(value: string): void {
```

Comment:
I view the Workspace Context as a public API for extensions(here under property editors, workspace views etc), so we would not like any of those to change the name. So in that perspective a setName method should not be available. :-)

It could be an internal one, like #setName but I would be fine by setting the value directly on the label, like this.#label.setValue(...)